src/lightning/fabric/cli.py

```diff
@@ -34,7 +34,18 @@
 _CLICK_AVAILABLE = RequirementCache("click")
 _LIGHTNING_SDK_AVAILABLE = RequirementCache("lightning_sdk")
 
-_SUPPORTED_ACCELERATORS = ("cpu", "gpu", "cuda", "mps", "tpu")
+_SUPPORTED_ACCELERATORS = ("cpu", "gpu", "cuda", "mps", "tpu", "auto")
+
+
+def _choose_auto_accelerator() -> str:
```

Comment:
so why do we have now the same function?

Reply:
I tried to import from https://github.com/Lightning-AI/pytorch-lightning/blob/c85660c0da7777c667309a3ef937276353aacc27/src/lightning/pytorch/trainer/connectors/accelerator_connector.py#L333
But there were some dependencies errors, one for `torchmetrics` on some runners.

Reply:
https://github.com/Lightning-AI/pytorch-lightning/actions/runs/15755765823/job/44410806075